### DISCUSSION TITLE
Content is visible on custom domain, but console says SSL error

### DISCUSSION THREAD

**Discussion by rkranendonk** [GitbookIO/community#637] [0. General bugs and questions] [answered]
My content is visible on https://docs.canvasmethod.org, but the Gitbook console for the space reports an SSL error. Is this a problem, and if so, how can I solve it?

**Comment by aikoii** [comment, marked as answer]
@rkranendonk I've just purged the CDN cache for your custom domain. You should now see the "Domain connected!" message 😊

**Comment by rkranendonk** [comment]
Yes! Thank you

**Comment by aikoii** [comment]
Great! I'll close this for now. Feel free to reach out if anything else comes up
